test/testUtils/quickBuilderUtil.js

```diff
@@ -2,9 +2,9 @@ const nodeUtils = require("../../src/common/utils");
 const EnviornmentBuilder = require("../../src/main_controller/EnvironmentBuilder");
 const CoreServer = require("../../src/core/core_server_mock/core_server");
 const path = require("path");
-const _B1Path = path.join(__dirname, "./id-l");
+const _B1Path = path.join(__dirname, "./id-l.json");
 const _B1Port = "10300";
-const _B2Path = path.join(__dirname, "./id-d");
+const _B2Path = path.join(__dirname, "./id-d.json");
```

Comment:
Also 3, remove the weird `./` and split `"./testUtils/id-d.json"` to `"testUtils", "id-d.json"`.